File: Back-end/HealthMindBackend/HealthMindBackend.Application/DTOs/PlanoSaudePacienteDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HealthMindBackend.Application.DTOs
{
    public class PlanoSaudePacienteDTO
    {
        public String PlanoSaudeId { get; set; }
        public String NumeroCarteirinha { get; set; }
        public DateTime DataValidade { get; set; }
    }
}

File: Front-end/health-mind-system-frontend/src/shared/types/dtos/PlanoSaudePaciente.dto.ts
import { PlanoSaudeDTO } from "./PlanoSaude.dto";

export interface PlanoSaudePacienteDTO {
    pacienteId?: string,
    planoSaudeId: string,
    numeroCarteirinha: string,
    dataValidade: string | Date,
    planoSaudeDTO?: PlanoSaudeDTO
};